### DISCUSSION TITLE
download large file from photoprisme with webdav

### DISCUSSION THREAD

**Discussion by Noneangel** [photoprism/photoprism#3417] [Q&A] [answered]
When connected from Windows 10 to the WEBdav server of photoprism using the [official documentation](https://docs.photoprism.app/user-guide/sync/webdav/#server-url), I cannot download files heavier than 50 MB.
Photoprism is installed via docker. I have set the `PHOTOPRISM_ORIGINALS_LIMIT` variable to `-1`. When I ran `photoprism config` from within the container, `originals-limit` is set to `-1`.
When I try to download / copy a big file, Windows return this error:
```
Error 0x800700DF: The file size exceeds the limit allowed and cannot be saved.
```
I can download the same file via the web UI of photoprism without any problem.

**Comment by lastzero** [comment, marked as answer]
From what I remember, this is a configuration value that you need to change in Windows (feel free to send a pull request to add this information to our documentation at docs.photoprism.app):

- https://docs.druva.com/Knowledge_Base/inSync/Troubleshooting/WebDAV_download_fails_with_file_size_exceeds__the_limit_error
- https://github.com/cryptomator/cryptomator/issues/1410

**Reply by Noneangel** [reply]
>feel free to send a pull request to add this information to our documentation at docs.photoprism.app

I will do it.
Thanks
